monai/application/engine/multi_gpu_supervised_trainer.py

```diff
@@ -50,8 +45,8 @@ def _default_eval_transform(x, y, y_pred):
 
 
 @monai.utils.export("monai.application.engine")
-def create_multigpu_supervised_trainer(net, optimizer, loss_fn, devices=None, non_blocking=False, 
```

Comment:
As I said elsewhere, I wouldn't change this name it clashes with that from Ignite. I can see people using one or the other in their code and we shouldn't introduce confusion.

Reply:
Hi @ericspod and @madil90 ,

OK, let's temporarily close and postpone this PR to when we organize all kinds of trainers.
Do you agree?
Thanks.

Reply:
Hi @Nic-Ma @ericspod I removed the original changes and added a small commit to resolve CI issues. Basically skip tests when no GPUs are available. @ericspod can you please review and merge if it looks good.